frontend/src/components/SigninPage.js

```diff
@@ -84,7 +94,7 @@ function SigninPage() {
                 </DialogContent>
                 <DialogActions>
                     {isSigninSuccess
-                        ? <Button component={Link} to="/signin">Log in</Button>
+                        ? <Button component={Link} to="/dashboard">Done</Button>
```

Comment:
Could you define a constant for dashboard and centralize this url as well? If you agree thats better design :D

Reply:
fixed